build/resolver/git/entrypoint.sh

```diff
@@ -180,7 +187,7 @@ pull() {
         }
 
         echo "Fetch $SCM_REVISION from origin"
-        git fetch -v --depth=1 origin $SCM_REVISION
+        git fetch -v ${GIT_DEPTH_OPTION} origin $SCM_REVISION
```

Comment:
Use `${GIT_DEPTH_OPTION:-}` to use default empty value if not variable set, otherwise

Reply:
PTAL